src/interface/src/app/map/map-manager.ts

```diff
@@ -702,11 +702,11 @@ export class MapManager {
       var htmlContent = '';
       htmlContent += '<div class=parentlegend>';
       htmlContent += '<div><b>Legend</b></div>';
-        for (let i = 0; i < entries.length; i++) {
```

Comment:
maybe add a comment on why we're going in reverse (e.g. we're expecting a certain order from Geoserver, etc.).

Reply:
Added a comment — does it seem clear enough?